test/run/ch-convert.bats

```diff
@@ -326,49 +326,59 @@ test_from () {
 
 @test 'ch-convert: --no-clobber' {
     # ch-image
-    printf 'FROM alpine:3.9\n' | ch-image build -t tmpimg -f - "$BATS_TMPDIR"
+    printf 'FROM alpine:3.17\n' | ch-image build -t tmpimg -f - "$BATS_TMPDIR"
     run ch-convert --no-clobber -o ch-image "$BATS_TMPDIR" tmpimg
     echo "$output"
     [[ $status -eq 1 ]]
     [[ $output = *"error: exists in ch-image storage, not deleting per --no-clobber: tmpimg" ]]
 
+    # convert ch_timg into ch-image format
+    ch-image delete timg | 'true'
+    if [[ $(file "$ch_timg") = *"symbolic link"* ]]; then
+        # symlink to squash archive
+        ch-convert -i squash -o ch-image "$ch_timg" timg
+    else
+        # directory
+        ch-convert -i dir -o ch-image "$ch_timg" timg
```

Comment:
Not DRY — set a variable for the input format and have the `ch-convert` command just once.